New package: KhepriTikZ v0.1.4

JuliaRegistrator opened the pull request:
- Registering package: KhepriTikZ
- Repository: https://github.com/aptmcl/KhepriTikZ.jl
- Created by: @aptmcl
- Version: v0.1.4
- Commit: 877666ec41f6a07aac529d9829c7332585c2d779
- Reviewed by: @aptmcl
- Reference: https://github.com/aptmcl/KhepriTikZ.jl/commit/877666ec41f6a07aac529d9829c7332585c2d779#commitcomment-149775291
<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd60a4a6bb0256581041e0cab5ffa0624b149150c7d5cb7c065eca97550af8ebcbbabffcb36c595c352f206179b964252956a0e50c6ac58c2e1292fd7551e244b2f0cb1fdf9b8f5095d67c527c6425a8a731ed2da9d5999db16f2b2a99642462f7af7a18fcb084295c6246762d7d39bbee2d3b1e00d3f07ed16d442b60d39ac6d802d40e98533a9eb469a08428e82dc1988289ec77f7bdd85019073fd4da75a380e998657f7bf443f6753584d826b686875 -->

goerz commented:
Thank you for submitting your package! Could you add a little bit of documentation before registering? [At the very least](https://discourse.julialang.org/t/when-does-it-make-sense-to-post-a-new-package-to-the-julia-repository/108701/4), that would be a description of the package's purpose and a small usage example in the README. An important part of packages in General is that any potential user can figure out what the package is about and how to get started with using it. That is really difficult when there is no documentation.

In the longer term, I definitely recommend setting up a [Documenter](https://documenter.juliadocs.org/stable/)-based documentation. Before a `v1.0` release, or for smaller packages that can be effectively described entirely with their `README`, that's not a requirement, though.

I would also recommend setting up the test suite in `tests/runtests.jl`.

JuliaRegistrator committed:
New package: KhepriTikZ v0.1.4

UUID: d7448f13-5788-4f2e-89b7-cfef8ec02b84
Repo: https://github.com/aptmcl/KhepriTikZ.jl.git
Tree: 59ebf5b576b29b2be6fb46242db76b2bd14d108f

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
